official/resnet/resnet_run_loop.py

```diff
@@ -602,7 +602,7 @@ def resnet_main(
       model_dir=flags_obj.model_dir,
       batch_size=flags_obj.batch_size)
 
-  def input_fn_train(num_epochs):
+  def input_fn_train(num_epochs, input_context=None):
```

Comment:
Looks like the lambda function in line 633 is not able to accept the input_context?

Reply:
Good catch!  I've fixed this now.